src/main/java/com/zuniorteam/racingcar/core/vo/GameResult.java

```diff
@@ -0,0 +1,29 @@
+package com.zuniorteam.racingcar.core.vo;
+
+import com.zuniorteam.racingcar.gui.drawer.PositionDrawer;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GameResult {
+
+    private final List<StepResult> stepResults;
+
+    public GameResult(List<StepResult> stepResult) {
+        assert stepResult != null;
+
+        stepResults = new ArrayList<>(stepResult);
```

Comment:
```suggestion
        this.stepResults = new ArrayList<>(stepResult);
```

여기서 `this`를 적지 않아도 알아서 잘 되는건가요? 🤔

Reply:
잉? 그러게요 이거 잘 동작하는건가요?

Reply:
헛, 실수인것같습니다,,,, ㅎㅎ... IDE에서 잡아주지않을까요?
저도 몰랐는데, 자세히보면 `stepResults`, `stepResult`로 나뉘어져있네요 ㅋㅋㅋ ....

Reply:
@cglee079 아 ㅋㅋㅋ 둘이 이름이 달랐군요....어쩐지....

Reply:
실수하기 쉬운 상태네요 ㄷㄷ